controllers/airline_controller_test.go

```diff
@@ -39,6 +42,34 @@ func TestHandleAirlineByIdController(t *testing.T) {
 	mockService.EXPECT().GetAirlineById(gomock.Any(), ctx, gomock.Any()).Return(&mockAirlines, nil)
 	ctx.Request, _ = http.NewRequest("GET", "airline/12332", nil)
 	controllerRepo.HandleGetAirlineById(ctx)
+	fmt.Println(ctx.Writer.Status())
 	assert.Equal(t, http.StatusOK, ctx.Writer.Status())
 
 }
+
+func TestHandleCreateNewAirline(t *testing.T) {
+
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+	mockService := mocks.NewMockIAirlineRepository(mockCtrl)
+	controllerRepo := NewAirlineControllerRepository(mockService)
+	recorder := httptest.NewRecorder()
+	ctx, _ := gin.CreateTestContext(recorder)
+	airline := models.Airlines{
+		Name: "XYZAirlines",
+	}
+	mockService.EXPECT().CreateNewAirline(gomock.Any(), ctx, &airline).Return(nil)
+	reqBody := `{"name":"XYZAirlines"}`
+
+	ctx.Request, _ = http.NewRequest("POST", "/airline", strings.NewReader(reqBody))
+	controllerRepo.HandleCreateNewAirline(ctx)
+	fmt.Println(ctx.Writer.Status())
```

Comment:
why do we need the `fmt.Println` call here?

Reply:
Used for debugging..forgot to remove...will remove it now